lib/msal-react/src/hooks/useMsalAuthentication.ts

```diff
@@ -48,10 +74,60 @@ export function useMsalAuthentication(
                 logger.verbose("useMsalAuthentication - Calling ssoSilent");
                 return instance.ssoSilent(loginRequest as SsoSilentRequest);
             default:
-                throw "Invalid interaction type provided.";
+                throw ReactAuthError.createInvalidInteractionTypeError();
         }
     }, [instance, interactionType, authenticationRequest, logger]);
 
+    const acquireToken = useCallback(async (callbackInteractionType?: InteractionType, callbackRequest?: SilentRequest): Promise<AuthenticationResult|null> => {
+        const fallbackInteractionType = callbackInteractionType || interactionType;
+
+        let tokenRequest: SilentRequest;
+
+        if (callbackRequest) {
+            tokenRequest = {
+                ...callbackRequest
+            };
+        } else if (authenticationRequest) {
+            tokenRequest = {
+                ...authenticationRequest,
+                scopes: authenticationRequest.scopes || OIDC_DEFAULT_SCOPES
+            };
+        } else {
+            tokenRequest = {
+                scopes: OIDC_DEFAULT_SCOPES
+            };
+        }
```

Comment:
Might be helpful to add some log messages in these if/else blocks.

Reply:
Done